# Set Points
point1: [0.0, 0.0, 4.0, 90]
point2: [4.0, 0.0, 4.0, 0]
point3: [4.0, 4.0, 4.0, 0]
point4: [0.0, 4.0, 4.0, 0]
